packages/accounts-base/accounts_server.js

```
@@ -146,9 +151,15 @@ export class AccountsServer extends AccountsCommon {
     // in a method or publish function is to do Meteor.find(this.userId).observe
     // and recompute when the user record changes.
-    const currentInvocation = DDP._CurrentMethodInvocation.get() || DDP._CurrentPublicationInvocation.get();
-    if (!currentInvocation)
-      throw new Error("Meteor.userId can only be invoked in method calls or publications.");
+    const currentInvocation =
+      DDP._CurrentMethodInvocation.get() ||
+      DDP._CurrentPublicationInvocation.get() ||
+      this._CurrentEndpointInvocation.get();
+    if (!currentInvocation) {
+      throw new Error(
+        "Meteor.userId can only be invoked in method, publications or WebApp endpoints calls."
```

ESLint

[error] Replace `"Meteor.userId·can·only·be·invoked·in·method,·publications·or·WebApp·endpoints·calls."` with `'Meteor.userId·can·only·be·invoked·in·method,·publications·or·WebApp·endpoints·calls.',`

(prettier/prettier)